ansible/monitoring.yml

```diff
@@ -39,7 +39,7 @@
         init_config:
         instances:
           - name: API
-            url: "http://localhost:3013/v2/top"
+            url: "{{ api_base_uri }}/top"
```

Comment:
I see defined in https://github.com/aeternity/infrastructure/blob/95c8d0dd616436ad5d8e11b2803d2d787e24b533/ansible/group_vars/epoch/vars.yml#L5